Tests/AspNetCore2.Tests/Saml2AuthExtensionsTests.cs

```diff
@@ -93,9 +93,10 @@ public void Saml2AuthExtensions_AddSaml2_RegisterSaml2Scheme_CustomDisplayName()
         [TestMethod]
         public void Saml2AuthExtensions_AddSaml2_NullCheckBuilder()
         {
-            AuthenticationBuilder builder = null;
+            var serviceCollection = new ServiceCollection();
+            var builder = new AuthenticationBuilder(serviceCollection);
 
-            builder.Invoking(b => b.AddSaml2(opt => { }))
+            builder.Invoking(b => ((AuthenticationBuilder)null).AddSaml2(opt => { }))
```

Comment:
fluentassert v6 has null argument check on invoking method this fixes it so that it still tests the null check in the AddSaml2 method